Marlin/example_configurations/BQ/Hephestos_2/Configuration_adv.h

```diff
@@ -44,7 +44,7 @@
 // Hephestos 2 24V heated bed upgrade kit.
 // https://store.bq.com/en/heated-bed-kit-hephestos2
 //
-//#define HEPHESTOS2_HEATED_BED_KIT
+#define HEPHESTOS2_HEATED_BED_KIT
```

Comment:
Best to leave this off by default since it's an optional, not standard, feature. Or is it?